examples/motoko_examples/simple-to-do/canisters/azle/index.ts

```diff
@@ -0,0 +1,66 @@
+import { Opt, Query, Update, ic, nat, nat64 } from 'azle';
+import { ToDo } from './types';
+
+import * as Utils from './utils';
+
+type PerfResult = {
```

Comment:
Let's group all of the performance stuff together at the top of the file, `PerfResult` then `perf_result` then `get_perf_result` then `record_performance`